specification/mediaservices/resource-manager/Microsoft.Media/stable/2022-08-01/streamingservice.json

```diff
@@ -698,6 +796,107 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{accountName}/liveOutputOperations/{operationId}": {
+      "get": {
+        "tags": [
+          "LiveOutputs"
+        ],
+        "summary": "Get operation status.",
+        "description": "Get a Live Output operation status.",
+        "operationId": "LiveOutputs_AsyncOperation",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/OperationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/AsyncOperationResult"
```

Comment:
How is this different from /v2/types.json#/definitions/ErrorResponse which we use for all other calls in the same file?

Reply:
I mean instead of AsyncOperationErrorDetail can we not use ErrorResponse

Reply:
AsyncOperationErrorDetail and ErrorResponse are two different types. They contain different properties. AsyncOperationErrorDetail is a part of response body of azure async operation endpoints while ErrorResponse is the general response body when any error occurs to any endpoint.